lib/cmd/internal/df.js

```diff
@@ -117,14 +120,16 @@ function logs(appName, cb) {
 }
 
 function stop(appName, cb) {
-  request(getDynoFarmUrl() + "stop/" + appName, function (err, response, body) {
+  var proxyRequest = request.defaults({'proxy': fhc.config.get("proxy")})
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)